### ISSUE TITLE
search bar doesn't works

### ISSUE THREAD

**Issue by Aditi-Singh16** [cdli-gh/cdli-mobile-app#19]
I searched for Montserrat, but the getdataFromAPI function returned an empty list
![image](https://user-images.githubusercontent.com/74586135/129491577-fd1965d0-58cc-41f9-8c71-4367ba6568b6.png)
search screen:
![WhatsApp Image 2021-08-16 at 1 49 40 AM](https://user-images.githubusercontent.com/74586135/129491625-44a8c6e8-32de-490c-b1c5-3dbf465b67d6.jpeg)

**Comment by epageperron** [comment]
Hi Aditi, thanks for reporting this! Unfortunately we don't have anyone in charge of the repo at this time.

**Comment by anila-a** [comment]
@Aditi-Singh16 the app should be fetching only the 10 newest artifacts from the API (with pagination), that's why it doesn't show up. However, this is a temporary limitation, as soon as the new framework is published we will set up the new API link to the app.

**Comment by Aditi-Singh16** [comment]
Oh okay👍🏻
Also , @anila-a  actually I made a few changes in the code like 
1. flatbutton ->  Textbutton
2. Raisedbutton -> Elevated button
3. Cach_image -> Cache network image
4. In platform widgets
     Android -> material
     iOS -> Cupertino


So, I did these changes because there was some version conflict error and then the app build was successful 
But now ,when I click on the drawer , it doesn't listen to the onClick event.
So what should I do now?
